frontend-react/index.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
same here

Reply:
Potentially autofixed from new project settings on save/autoformat, not an issue.